pkg/auth/handler/webapp/viewmodels/authflow_branch.go

```diff
@@ -106,6 +106,8 @@ func newAuthflowBranchViewModelStepAuthenticate(screen *webapp.AuthflowScreenWit
 			addChannelBranch(idx, o)
 		case config.AuthenticationFlowAuthenticationSecondaryOOBOTPSMS:
 			addChannelBranch(idx, o)
+		case config.AuthenticationFlowAuthenticationSecondaryTOTP:
```

Comment:
:shocked: looks like a bug exist in main too

Reply:
Yes, I will create another pr to main with this commit after.

Reply:
I suspect we will catch more of these later on, but seems not worth trying to handle in https://github.com/authgear/authgear-server/issues/3653 🤔

Reply:
Base branch changed to main